github_pr.py

```diff
@@ -149,6 +149,40 @@ def _return_specific_comment_prs(all_prs, filters):
         return []
 
 
+def _check_approved_mergers_file(approved_users, comment_users):
+    if not isinstance(approved_users, list):
```

Comment:
`_check_approved_mergers_file()` should always take a file as an argument. `_check_approved_mergers()` should always take a list as an argument. `_check_approved_mergers_file()` would open the file and create a list then pass it to `_check_approved_mergers()`